samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,186 @@
+        Action Items in bullet points"
+        LocalTimeZone=Asia/Kolkata
+        APPINSIGHTS_INSTRUMENTATIONKEY=""
+        APPINSIGHTS_CONNECTIONSTRING=""
+    ``` 
+1) Create a policy for a demo tenant user for creating the online meeting on behalf of that user using the following PowerShell script (Images for reference only)
+ 
+      PowerShell script
+
+    ```powershell
+    # Import-Module MicrosoftTeams
+    # Call Connect-MicrosoftTeams using no parameters to open a window allowing for MFA accounts to authenticate
+    Connect-MicrosoftTeams
+    New-CsApplicationAccessPolicy -Identity “<<policy-identity/policy-name>>” -AppIds "<<microsoft-app-id>>" -Description "<<policy-description>>"
+    Grant-CsApplicationAccessPolicy -PolicyName “<<policy-identity/policy-name>>” -Identity "<<object-id-of-the-user-to-whom-the-policy-needs-to-be-granted>>"
+    
+    OR
+    # For global access
+    # Grant-CsApplicationAccessPolicy -PolicyName Meeting-policy-dev -Global
+    ```
+    Example:
+
+    ```powershell
+      # Import-Module MicrosoftTeams
+      Connect-MicrosoftTeams
+
+      New-CsApplicationAccessPolicy -Identity Meeting-policy-dev -AppIds "d0bdaa0f-8be2-4e85-9e0d-2e446676b88c" -Description "Online meeting policy - contoso town"
+      
+      Grant-CsApplicationAccessPolicy -PolicyName Meeting-policy-dev -Identity "782f076f-f6f9-4bff-9673-ea1997283e9c"
+      OR
+      # For global access
+      # Grant-CsApplicationAccessPolicy -PolicyName Meeting-policy-dev -Global
+    ```
+    
+     ![PolicySetup](Images/Policy.png)
+     
+1) In Azure [App Registration](https://ms.portal.azure.com/) Under left menu, navigate to **API Permissions**, and make sure to add the following permissions of Microsoft Graph API > Application permissions:
+
+    - `Calendars.Read`
+    - `Calendars.ReadBasic.All`
+    - `Calendars.ReadWrite`
+    - `EventListener.Read.All`
+    - `EventListener.ReadWrite.All`
+    - `OnlineMeetingArtifact.Read.All`
+    - `OnlineMeetingRecording.Read.All`
+    - `OnlineMeetings.Read.All`
+    - `OnlineMeetings.ReadWrite.All`
+    - `OnlineMeetingTranscript.Read.All`
+    - `User.Read.All`
+    - `User.ReadBasic.All`
+    - `User.ReadWrite.All`
+
+        ![Application Permission](Images/ApplicationPermission.png)
+
+1) Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+
+## Try it yourself - experience the App in your Microsoft Teams client
+Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
```

Comment:
This step should come on top.
Refer Hari PR>

Reply:
Done